rest-client-base/src/main/java/com/wultra/core/rest/client/base/DefaultRestClient.java

```diff
@@ -644,9 +642,9 @@ public <T> ObjectResponse<T> headObject(String path, MultiValueMap<String, Strin
      * @return Parameterized type reference of ObjectResponse.
      */
     private <T> ParameterizedTypeReference<ObjectResponse<T>> getTypeReference(Class<T> responseType) {
-        return new ParameterizedTypeReference<ObjectResponse<T>>(){
+        return new ParameterizedTypeReference<>() {
             @Override
-            public Type getType() {
+            public @NotNull Type getType() {
```

Comment:
Is that needed for Spring Boot upgrade to 3? It means a new dependency `org.jetbrains.annotations`.